docs/zh/start/file/minio.md

```diff
@@ -0,0 +1,48 @@
+# 基于MinIO OSS实现文件的增删改查
+
+## 快速开始
+
+Layotto提供了访问文件的示例 [demo](../../../../demo/file/client.go),该示例实现了文件的增删改查操作。
```

Comment:
We should use absolute path here,e.g. https://github.com/mosn/layotto/blob/main/demo/file/client.go
Since this document will be opened like "https://mosn.io/layotto/#/zh/start/file/start" and the hostname will be "mosn.io" instead of "github.com" , this `client.go` won't be found under "mosn.io".

There is a document for it,see https://mosn.io/layotto/#/zh/development/contributing-doc?id=%e7%bb%8f%e5%b8%b8%e9%81%87%e5%88%b0%e7%9a%84%e5%9d%91%ef%bc%9a%e6%96%87%e6%a1%a3%e5%86%85%e7%9a%84%e8%b6%85%e9%93%be%e6%8e%a5

In fact,I just found that the document for alicoud oss is wrong ,too :(